### PR TITLE
feat(curriculum): add English curriculum block 6.3 clarification on code understanding

### PR DESCRIPTION
**Pull request by miyaliu666**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-for-clarification-on-code-understanding/65f59409f39a43b8d90b53f7.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+dashedName: task-36
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE: 
+Brian: No problem, Sophie. I can definitely work with you on that. Let's start by understanding the symptoms of the issue.
+-->
+
+# --description–-
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-for-clarification-on-code-understanding/65f59409f39a43b8d90b53f7.md:14]:
```suggestion
# --description--
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Tiny little detail here broke the tests

**Comment by miyaliu666** [comment]
@naomi-lgbt Thanks for reviewing the PRs. Should I update the status label after applying the review suggestions?

**Review by moT01 — CHANGES_REQUESTED** [review]
This looks good @miyaliu666 🎉 

Do you mind removing the `audioPath` from all the challenge files?

**Review by moT01 — APPROVED** [review]
LGTM 🎉

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-for-clarification-on-code-understanding/65f52fb434a8875cb666ede5.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+<!--
+AUDIO REFERENCE: 
+Sarah: Where does it seem to be happening?
+-->
+
+# --description--
+
+`seem to be` is used when you are not sure about something and you want to guess or suggest a possibility. For example, if you say `It seems to be working`, you are guessing that something is working but you are not 100% sure.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-for-clarification-on-code-understanding/65f52fb434a8875cb666ede5.md:15]:
```suggestion
`Seem to be` is used when you are not sure about something and you want to guess or suggest a possibility. For example, if you say `It seems to be working`, you are guessing that something is working but you are not 100% sure.
```

If it's at the beginning of a sentence it should be capitalised.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**